src/elements/content-sidebar/additional-tabs/AdditionalTab.js

```diff
@@ -12,9 +12,9 @@ import { bdlGray50 } from '../../../styles/variables';
 import PlainButton from '../../../components/plain-button/PlainButton';
 import AdditionalTabTooltip from './AdditionalTabTooltip';
 import AdditionalTabPlaceholder from './AdditionalTabPlaceholder';
-import Apps16 from '../../../icon/fill/Apps16';
 import messages from './messages';
 import type { AdditionalSidebarTab, AdditionalSidebarTabFtuxData } from '../flowTypes';
+import IconEllipsis from '../../../icons/general/IconEllipsis';
```

Comment:
there's an opportunity to import the blueprint equivalent icon directly since the `src/icons/` folder will eventually be deprecated in favor of blueprint-assets